src/__tests__/__snapshots__/transpiled-code.ts.snap

```diff
@@ -0,0 +1,514 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Empty code returns empty string 1`] = `
+Object {
+  "code": "",
+  "transpiled": "",
+}
+`;
+
+exports[`Ensure return values are of the form {isTail...} after function declarations are transformed 1`] = `
+Object {
+  "code": "function f() {
+  return null;
+}",
+  "transpiled": "const native = $$NATIVE_STORAGE;
```

Comment:
Actually I'm starting to think that these transpiled snippets are small enough that I can live with adding these to all the native-enabled tests... What do you think?

Reply:
Yes this does sound reasonable.